deploy/kubernetes/base/resources.yaml

```diff
@@ -42,7 +42,9 @@ rules:
   - apiGroups: [ "" ]
     resources: [ "endpoints" ]
     verbs: [ "get", "list", "watch", "create", "update", "patch" ]
-
+  - apiGroups: [ "apps" ],
+    resources: [ "daemonsets" ]
+    verbs: [ "get", "list" ],
```

Comment:
No comma, you can execute `deploy/k8s.yaml` to generate `k8s.yaml` and `k8s_before_v1_18.yaml` from `base/resources.yaml` automatically.

Reply:
Fixed. by the way, I failed to run `make deploy/k8s.yaml` as I am using kustomize v5.0.0, seems the `kustomization.conf` is not compatible with it.

```text
echo "# DO NOT EDIT: generated by 'kustomize build'" > deploy/k8s.yaml
kustomize build deploy/kubernetes/release >> deploy/k8s.yaml
Error: invalid Kustomization: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go struct field Kustomization.patches of type types.Patch
make: *** [deploy/k8s.yaml] Error 1
```

Reply:
I have fixed it in https://github.com/juicedata/juicefs-csi-driver/pull/590. Can you merge latest master branch and try again?

Reply:
It works now:

```
$ make deploy/k8s.yaml
echo "# DO NOT EDIT: generated by 'kustomize build'" > deploy/k8s.yaml
kustomize build deploy/kubernetes/release >> deploy/k8s.yaml
cp deploy/k8s.yaml deploy/k8s_before_v1_18.yaml
sed -i.orig 's@storage.k8s.io/v1@storage.k8s.io/v1beta1@g' deploy/k8s_before_v1_18.yaml
```

But no difference to commit, the generated files are not changed.